README.md

```diff
@@ -257,32 +203,18 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
```

Comment:
[OPTIONAL] 
- [ ] please Kindly remove these texts as they only serve descriptive purposes which are to guide you on how to customise this section of your readme file, once customized to your taste as you have done you do not need them anymore and you can take it off :+1: